src/replication.c

```diff
@@ -1832,6 +1904,10 @@ static int useDisklessLoad(void) {
     return enabled;
 }
 
+static int useRdbChannelSync(void) {
+    return server.rdb_channel_enabled && (server.primary_can_sync_using_rdb_channel == 1);
```

Comment:
```suggestion
    return server.rdb_channel_enabled && server.primary_can_sync_using_rdb_channel;
```
was there a reason for the `== 1` part?
it's generally better to do `!=0` rather than `==1`